playbooks/azure/openshift-cluster/files/get-node-logs

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -x
+
+LOG_DIR=$(mktemp -d)
+
+trap "{ rm -rf $LOG_DIR ; exit 1; }" INT TERM ERR
```

Comment:
`trap "{ rm -rf $LOG_DIR; }" EXIT` ?

Reply:
This means it will delete all logs on exit (even archive). In this case "cleaning dump" happens only if it fails. Exit clean results recursively of what it just gathered.

Reply:
You're not creating the archive in $LOG_DIR so it won't be lost.